checkout-ui-custom/src/_js/_v.custom.checkout.ui.js

```diff
@@ -976,10 +976,11 @@ class checkoutCustom {
 
     if (_this.customAddressForm && _orderForm.canEditData) {
       $('body').removeClass('returningUser')
-      _this.customAddressForm.init(_orderForm)
     } else {
       $('body').addClass('returningUser')
     }
+
+    _this.customAddressForm.init(_orderForm)
```

Comment:
Cypress says this line is throwing an error: `this.customAddressForm.init is not a function`

I think it needs to be inside that conditional (`if (_this.customAddressForm) ...`)